.github/actions/setup_test_environment/action.yml

```diff
@@ -1,20 +1,24 @@
 name: "Setup test environment"
 
+description: Setup ROCm test environment from built artifacts
```

Comment:
So it turns out `type` is not allowed in these reusable workflows for inputs, and "description" is required?

I'm not sure where the docs say that, but I referenced pytorch's extensive actions folder: https://github.com/pytorch/pytorch/tree/main/.github/actions